tests/arp/test_arp_dualtor.py

```diff
@@ -63,17 +63,14 @@ def pause_arp_update(duthosts):
 
 
 @pytest.fixture(params=['IPv4', 'IPv6'])
-def neighbor_ip(request, mux_config):       # noqa: F811
-    """
-    Provide the neighbor IP used for testing
-
-    Randomly select an IP from the server IPs configured in the config DB MUX_CABLE table
-    """
+def selected_mux_port(request, mux_config):       # noqa: F811
+    """Randomly select a mux port for testing."""
     ip_version = request.param
     selected_intf = random.choice(list(mux_config.values()))
     neigh_ip = ip_interface(selected_intf["SERVER"][ip_version]).ip
+    cable_type = selected_intf["SERVER"].get("cable_type", "active-standby")
```

Comment:
Does this change also work on single ToR simulated dualtor?

Reply:
this testcase is only for dualtor testbed.